src/productcatalogservice/product_catalog.go

```diff
@@ -62,7 +62,6 @@ func (p *productCatalog) GetProduct(ctx context.Context, req *pb.GetProductReque
 func (p *productCatalog) SearchProducts(ctx context.Context, req *pb.SearchProductsRequest) (*pb.SearchProductsResponse, error) {
 	time.Sleep(extraLatency)
 
-	// Interpret query as a substring match in name or description.
```

Comment:
**Question (just curious, no change needed):** Did you remove this comment because it's false or because it feels unnecessary?

Reply:
The 3 lines were already self-explanatory